frontend/pages/hosts/ManageHostsPage/ManageHostsPage.jsx

```diff
@@ -81,39 +81,33 @@ export class ManageHostsPage extends Component {
     };
   }
 
-  componentWillMount () {
-    const { dispatch } = this.props;
-
-    dispatch(hostActions.loadAll());
-    this.buildSortedHosts();
-
-    return this.getEntities();
-  }
-
   componentDidMount () {
     const { getEntities } = this;
+    const { hosts } = this.props;
 
     this.interval = global.window.setInterval(getEntities, 5000);
```

Comment:
Should this setInterval be replaced with a setTimeout on completion?

Reply:
I took advantage of my current familiarity with this code to make this change.